<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1259</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>325</width>
         <height>793</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_7">
          <property name="title">
           <string>系统参数</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="1" column="0">
            <widget class="QLabel" name="label_44">
             <property name="text">
              <string>本机端口号：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_47">
             <property name="text">
              <string>本机IP地址：</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="rBtn_radarType">
             <property name="text">
              <string>海洋雷达</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_localPort"/>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_localIP"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>激光参数</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="5" column="0">
            <widget class="QRadioButton" name="radioButton">
             <property name="text">
              <string>RadioButton</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>激光重频：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_laser_freq"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_7"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>触发方式：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>激光功率：</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QRadioButton" name="radioButton_2">
             <property name="text">
              <string>RadioButton</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QPushButton" name="pushButton_laserInfo">
             <property name="text">
              <string>详细信息</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>电机参数</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>电机转速（rpm/s)：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_8"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>高压参数</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="2">
            <widget class="QLineEdit" name="lineEdit_9"/>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>DA0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>DA1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>DA2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>DA3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>DA4</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>      DA通道：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>        值：</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>读取</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>        值：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>设置</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>      AD通道：</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="comboBox_2">
             <item>
              <property name="text">
               <string>DA0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>DA1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>DA2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>DA3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>DA4</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLineEdit" name="lineEdit_10"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>预览参数</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit_firstStartPos"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>抽样率：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_sampleRate"/>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEdit_firstLen"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_sumThreshold">
             <property name="text">
              <string>和阈值：</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lineEdit_secondStartPos">
             <property name="toolTip">
              <string>第二段起始位置需要大于第一段提取位置和提取长度之和</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_subThreshold">
             <property name="text">
              <string>值阈值：</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QPushButton" name="pushButton_sampleEnable">
             <property name="text">
              <string>开始采集</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="lineEdit_sumThreshold"/>
           </item>
           <item row="7" column="1">
            <widget class="QLineEdit" name="lineEdit_subThreshold"/>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="lineEdit_secondLen">
             <property name="toolTip">
              <string/>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_secondLen">
             <property name="text">
              <string>第二段提取长度：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>采样长度：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_sampleLen"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>第一段提取位置：</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>第一段提取长度：</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_secondStartPos">
             <property name="text">
              <string>第二段提取位置：</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QPushButton" name="pushButton_setPreviewPara">
             <property name="text">
              <string>设置</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>文件参数</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_43"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_45">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>文件名：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_6">
             <property name="text">
              <string>检索文件</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButton_7">
             <property name="text">
              <string>详细信息</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="ChartView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1259</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionNote"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>关于</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSetting"/>
   <addaction name="actionLaserSetting"/>
   <addaction name="actionMotorSetting"/>
   <addaction name="actionSampleSetting"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSetting">
   <property name="text">
    <string>系统设置</string>
   </property>
   <property name="toolTip">
    <string>设置系统参数</string>
   </property>
  </action>
  <action name="actionLaserSetting">
   <property name="text">
    <string>激光器设置</string>
   </property>
  </action>
  <action name="actionMotorSetting">
   <property name="text">
    <string>电机设置</string>
   </property>
  </action>
  <action name="actionSampleSetting">
   <property name="text">
    <string>抽样设置</string>
   </property>
  </action>
  <action name="actionNote">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>操作注意事项</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">./inc/chartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
